float64[16] pose
uint64 frame_id